File: IntegrationTests/CocoaPods-iOS-static/ExampleTests/ExampleTests.m
//
//  ExampleTests.m
//  ExampleTests
//
//  Created by Victor Ilyukevich on 5/21/16.
//  Copyright © 2016 Open Source. All rights reserved.
//

#import <XCTest/XCTest.h>
#import "AppDelegate.h"

@interface ExampleTests : XCTestCase

@end

@implementation ExampleTests

- (void)testFoo {
  AppDelegate *a = [AppDelegate new];
  XCTAssertEqualObjects(a.foo, @"vor 1 Stunde");
}

- (void)testNumberOfLocalizations {
  XCTAssertEqual([NSBundle mainBundle].localizations.count, 2);
}

@end

File: IntegrationTests/CocoaPods-tvOS/ExampleTests/ExampleTests.swift
//
//  ExampleTests.swift
//  ExampleTests
//
//  Created by Victor Ilyukevich on 5/21/16.
//  Copyright © 2016 Open Source. All rights reserved.
//

import XCTest
@testable import Example

class ExampleTests: XCTestCase {
  func testFoo() {
    let a = AppDelegate()
    XCTAssertEqual(a.foo(), "hace 3 semanas")
  }

  func testNumberOfLocalizations() {
    XCTAssertEqual(Bundle.main.localizations.count, 2)
  }
}
